setup.js

```diff
@@ -0,0 +1,70 @@
+
+//azure config mode arm
+var uuid = require('node-uuid');
+var fs = require('fs');
+var spawn = require('child_process').spawn;
+
+var uuidrg = uuid.v4().toLowerCase().replace(/-/g, '');
+var resourceGroup = "rg" + uuidrg.substring(0,19);
+var storageAccountName = "sa" + uuidrg.substring(0,19);
+var location = "westus";
+var errorHasOccurred = false;
+var connectionData = "";
+var wasInArmMode = false;
+
+executeAzureCLI(['config', 'list'], function() {
+	executeAzureCLI(['config', 'mode', 'arm'], function() {
+		executeAzureCLI(['group', 'create', resourceGroup, location], function() {
+			executeAzureCLI(['storage', 'account', 'create', storageAccountName, '-g', resourceGroup, '--type', 'RAGRS', '-l', location], function(code) {
```

Comment:
We just added the support for encryption@rest and Cool storage om CLI. With the change, '--type' is renamed to '--sku-name' and the '--kind' parameter is added (set to "Storage" in your case).